### ISSUE TITLE
Feature Request : Integrate In-app update

### ISSUE THREAD

**Issue by Ni3verma** [Ni3verma/Safe-Box#85]
**Is your feature request related to a problem? Please describe.**
User has to manually go to play store to update app (if auto update is off)

**Describe the solution you'd like**
Implement in-app updates so that users are always on latest version
https://developer.android.com/guide/playcore/in-app-updates

**Comment by Ni3verma** [comment]
@AdiAr11 sure, before starting, please read [contribution ](https://github.com/Ni3verma/Safe-Box/blob/master/CONTRIBUTING.md)guidelines

**Comment by Ni3verma** [comment]
Hi @AdiAr11 , are you working on it?

**Comment by AdiAr11** [comment]
@Ni3verma yes I'm sorry i got a little busy during this festive season.. I'll complete this task asap

**Comment by AdiAr11** [comment]
@Ni3verma could you share the slack workspace link again please? Because the previous one is no longer active. Also, I've implemented the in-app update functionality but I'm can't test its functionality.

**Comment by Ni3verma** [comment]
@AdiAr11 updated the link
sure I will see your PR
